terraform/environments/bootstrap/delegate-access/iam.tf

```diff
@@ -179,6 +179,166 @@ resource "aws_iam_policy" "member-access" {
   policy      = data.aws_iam_policy_document.member-access.json
 }
 
+module "instance-scheduler-access" {
+  count  = local.account_data.account-type == "member" && terraform.workspace != "testing-test" ? 1 : 0
+  source = "github.com/ministryofjustice/modernisation-platform-terraform-cross-account-access?ref=v2.2.0"
+  providers = {
+    aws = aws.workspace
+  }
+  account_id             = local.modernisation_platform_account.id
+  additional_trust_roles = [format("arn:aws:iam::%s:role/github-actions", local.environment_management.account_ids[terraform.workspace])]
```

Comment:
I think the additional_trust_roles would need to give permissions to the lambda running the scheduler in the shared services account, is that right @dms1981 ?

Reply:
The goal for this ticket should be that the lambda function has a role that allows it to assume a role in a separate account (or, I suppose, in the same account).
That assumable role should be the one that has both the permissions to start / stop instances, and a trust relationship statement allowing the lambda function to assume the role.
So... @davidkelliott , yes. and @gfou-al , also yes. But the plan is, in future, to run the lambda from `core-shared-services`.